node/task.ts

```diff
@@ -2030,15 +2024,15 @@ if (semver.lt(process.versions.node, '4.2.0')) {
 // Populate the vault with sensitive data.  Inputs and Endpoints
 //-------------------------------------------------------------------
 
-// only exposed as a function so unit tests can reload vault for each test
+// only exposed as a function so unit tests can reload vault for each test.
 // in prod, it's called globally below so user does not need to call
 
-export function _loadData(): void {
+function _loadData(): void {
```

Comment:
`_loadData` is for internal unit testing only, correct? otherwise it needs to continue to be exported